docs/upgrading/future.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Future Flags
```

Comment:
We had a dead link to this already in the API development strategy doc, so added a shell here we can use once we have a stable flag